inc/saitypes.h

```diff
@@ -463,6 +465,9 @@ typedef struct _sai_tunnel_map_params_t
     /** VNI id */
     sai_uint32_t vni_id;
 
+    /** Bridge IF */
+    sai_object_id_t bridge_if;
```

Comment:
The idea is to support 2 new types of maps - VNI to Bridge IF, and Bridge IF to VNI
This requires adding bridge_if to the map params
There can be many map elements
Not sure I understand the comment - "there is already oid in the strcut". Can you explain how you suggest to do it ?